index.html

```diff
@@ -40,15 +38,13 @@
         top: 0;
       }
     }
-
-    .dashboard_item {
+   .dashboard_item {
```

Comment:
If you have time, check out the Trello CSS style guide (https://gist.github.com/bobbygrace/9e961e8982f42eb91b80) and BEM (https://seesparkbox.com/foundry/bem_by_example). We use most of the Trello guide, but for components and modifiers we follow the BEM structure.

There's no need to change your CSS for them right now, but it might provide a guidance of how to write CSS effectively